src/batchUploader.ts

```diff
@@ -10,6 +10,7 @@ import { convertEvents } from './sdkToEventsApiConverter';
 import Types from './types';
 import { getRampNumber, isEmpty } from './utils';
 import { SessionStorageVault, LocalStorageVault } from './vault';
+import { AsyncUploader, FetchUploader, XHRUploader, fetchPayload } from './uploaders';
```

Comment:
I think the linter should have split this line up for you.

Reply:
We'll need to get our linters aligned one day.  maybe a sustained engineering session.  mine seems to work with js but not ts, and freak out sometimes.  i'll update for now though.

Reply:
I already have one on the books: https://go.mparticle.com/work/SQDSDKS-5147